doc/network-forwards.md

```diff
@@ -1,59 +1,96 @@
 # How to configure network forwards
 
-The following network types support forwards. See each network type section for more details.
+```{note}
+Network forwards are available for the {ref}`OVN network <network-ovn>` and the {ref}`Bridge network <network-bridge>`.
+```
 
- - [bridge](#network-bridge)
- - [ovn](#network-ovn)
+Network forwards allow an external IP address (or specific ports on it) to be forwarded to an internal IP address (or specific ports on it) in the network that the forward belongs to.
 
- Network forwards allow an external IP address (or specific ports on it) to be forwarded to an internal IP address (or specific ports on it) in the network that the forward belongs to.
+This feature can be useful if you have only one public IP address, but you don't want to statically assign this address to one instance only.
+There are two different ways how you can use network forwards in this case:
+
+- Forward all traffic from the external address to the internal address of one instance.
+  This method makes it easy to move the external address to another instance by simply reconfiguring the network forward.
```

Comment:
Similar to "assign" above, this does not make it possible to move an external address to another instance (the external address is never configured/assigned inside the instance), but instead it makes it easy to move traffic destined to the external IP address to another instance (the actual assignment of the external address never changes - it is always bound on the LXD network).